README.md

```diff
@@ -1741,4 +1742,4 @@ API | Description | Auth | HTTPS | CORS |
 <br>
 
 ## License
-[MIT](LICENSE)
+[MIT](LICENSE)
```

Comment:
IMHO you should not remove the newline at end of file

Reply:
Oh, my bad! I didn't notice the newline had been removed, probably done by some setting on Visual Studio Code. I'll fix that asap!